pip/req/req_uninstall.py

```diff
@@ -47,7 +47,7 @@ def _can_uninstall(self):
         return True
 
     def add(self, path):
-        path = normalize_path(path)
+        path = os.path.normcase(os.path.expanduser(path))
```

Comment:
I called it `resolve_symlinks`. I considered realpath, but it seemed confusing to have parameters called 'path' and 'realpath'. I'm happy to change this to another name if you prefer.